temporian/implementation/numpy/data/sampling.py

```diff
@@ -4,15 +4,18 @@
 
 
 class NumpySampling:
-    def __init__(self, names: List[str], data: Dict[Tuple, np.ndarray]) -> None:
-        self.names = names
+    def __init__(self, index: List[str], data: Dict[Tuple, np.ndarray]) -> None:
+        self.index = index
         self.data = data
 
+    def __repr__(self) -> str:
+        return f"index:{self.index} data: {self.data.__repr__()}"
+
     def __eq__(self, other):
```

Comment:
Nit: add `__eq__` input and output return types, i.e. `other : Any` and `-> bool`.

Reply:
Done